### DISCUSSION TITLE
openresty-voms container security upgrade?

### DISCUSSION THREAD

**Discussion by zhangxt-ihep** [indigo-iam/iam#749] [Q&A]
Hi,
  It seems that `indigoiam/openresty-voms` container has no upgrade in 3 years and its dependency container `storm2/openresty-voms` has no upgrade in last 2 years. Our network security team reports openresty with version <1.15.8.4 has security risks. Is it possible to upgrade the openresty version in official docker container and make a regular container rebuilding CI/CD for it? Thank you!

**Comment by federicaagostini** [comment]
Hi, which is the usage of this container? If it proxies requests to voms-aa you can use the nginx-voms image: `baltig.infn.it:4567/cnafsd/ngx_http_voms_module/nginx-httpg-voms`. An example of its configuration is [here](https://github.com/indigo-iam/iam/blob/master/compose/voms-deploy/assets/nginx/conf.d/voms.test.example.conf).

**Reply by giacomini** [reply]
Which is also pushed to Docker Hub https://hub.docker.com/r/cnafsd/nginx-httpg-voms

**Reply by zhangxt-ihep** [reply]
Thank you. Is `indigoiam/openresty-voms` no longer updated? I followed the [documentation](https://indigo-iam.github.io/v/v1.8.4/docs/getting-started/docker/) and seems not using `cnafsd/nginx-httpg-voms` as nginx's container.

**Comment by federicaagostini** [comment]
The correct doc page is [this one](https://indigo-iam.github.io/v/current/docs/tasks/deployment/voms/), but yes, it still refers to openresty-voms, so we are going to update it right now.
